IBAnimatable/FillDesignable.swift

```diff
@@ -34,6 +37,62 @@ public extension FillDesignable where Self: UITableViewCell {
     if let unwrappedFillColor = fillColor {
       backgroundColor = unwrappedFillColor
       contentView.backgroundColor = unwrappedFillColor
+    } else if let unwrappedPredefinedColor = predefinedColorFromString(predefinedColor) {
+      backgroundColor = unwrappedPredefinedColor
+      contentView.backgroundColor = unwrappedPredefinedColor
     }
   }
 }
+
+private extension FillDesignable {
+  
+  func predefinedColorFromString(predefinedColor: String?) -> UIColor? {
+    guard let unwrappedColorTypeString = predefinedColor, colorType = ColorType(rawValue: unwrappedColorTypeString) else {
+      return nil
+    }
+    
+    switch colorType {
+    case .Emerland:
```

Comment:
@tbaranes Sorry, I am not sure are we on the same page? I mean for all flat colors, can we add a prefix for all of them like `FlatEmerland` and `FlatPomegranate` instead of `Emerland` and `Pomegranate`. 

The advantage of add `Flat` as prefix, the user knows they are from flat colors (https://gist.githubusercontent.com/tkrugg/a577c32e93eecc6c7991/raw/f7866132e26f9bffcda9caf13dced55e4a99e145/flatuicolors.json). And when we add other set of colors, e.g. Material Design color (https://www.google.com/design/spec/style/color.html#color-color-palette), then all of them will prefix as `Material`, e.g.  `MaterialRed500` and  `MaterialRed50`, those names may not be the final one, just an example to describe the usage. Another benefit is when the user uses the enum in code, when they can start typing `.Flat` and Xcode will auto complete the rest for them.

But there are disadvantage like getting the name longer than it should be.

Please let me know what you think.